clickfile.py

```diff
@@ -195,31 +216,53 @@ def run_oz_file(file_name):
         env["NETWORK_ID"] = str(network_manager.get_network_param(network, "network_ids.neon"))
         env["PROXY_URL"] = network_manager.get_network_param(network, "proxy_url")
 
+        start_time = time.time()
         out = subprocess.run(
             f"npx hardhat test {file_name}",
             shell=True,
             cwd=cwd,
             capture_output=True,
             env=env,
         )
+        end_time = time.time()
         stdout = out.stdout.decode()
         stderr = out.stderr.decode()
+        time_info = time_measure(start_time=start_time, end_time=end_time, job_name=file_name)
         print(f"Test {file_name} finished with code {out.returncode}")
         print(stdout)
         print(stderr)
+        print(time_info)
         keys_env.append(keys)
         log_dirs = cwd.parent / "results" / file_name.replace(".", "_").replace("/", "_")
         log_dirs.mkdir(parents=True, exist_ok=True)
         with open(log_dirs / "stdout.log", "w") as f:
             f.write(stdout)
         with open(log_dirs / "stderr.log", "w") as f:
             f.write(stderr)
+        with open(log_dirs / "time.log", "w") as f:
+            f.write(time_info)
 
+    print("Run priority tests first in parallel")
+    pool = Pool(len(priority_tests))
```

Comment:
You have a two pools, priority will run first in len(priority_tests) threads and only after this you will run remaining files. Hmm, I see how we create a list with tests and it's my fault. I thought we are running 8 jobs in parallel with a list of tests but I see that we run each test file separately. I think split to priority tests and simple tests doesn't affect time.
Could you pls remove this split and run OZ in 16 threads?

Reply:
Correct, but with help of separate pools and dedicated processes we can exclude situations, when two or three heavy tests are set after each other in a queue in one process. I suppose it could be the reason why sometimes instead of 1.5 hour we have the builds above 2 hours.

I can remove it for sure, and I did the build with 16 threads before with that configuration, for example:
https://github.com/neonlabsorg/neon-tests/actions/runs/8374357131/job/22929634331

Reply:
@lyubenkov You are right, but will better if you will use one pool and just move this 3 long tests to beginning of the list.